style.css

```
@@ -47,6 +47,10 @@ body {
   background-color: #786b59;
 }
 
+.isPressed {
```

```suggestion
.toolBtn.pressed {
```
Без указания второго класса может возникнуть путаница.
Название `isPressed` немного нарушает логику названия класса, так как это не булево значение, оно должно показывать принадлежность к какой-то группе, в данном случае - группа нажатых кнопок, поэтому `pressed` звучит логичнее и правильнее (в скрипте тогда тоже надо не забыть исправить)